hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconDeadNodeHandler.java

```diff
@@ -79,6 +85,7 @@ public void onMessage(final DatanodeDetails datanodeDetails,
       }
       containerHealthTask.triggerContainerHealthCheck();
       pipelineSyncTask.triggerPipelineSyncTask();
+      containerSizeCountTask.process(containerManager.getContainers());
```

Comment:
Have we verified if a node becomes dead, how container size will be impacted. Container may get closed on that DN, but some other replica for that container may be present in other DN in cluster. Pls check if we really need to call our task for DeadNode handler.

Reply:
For DeadDatanodeHandler we will be calling **destroyPipelines, closeContainers and removeContainerReplicas methods.** Hence since the container replicas are removed, the container size counts will also be affected. Therefore, we need to trigger the ContainerSizeCountTask.

Reply:
this means that the ContainerSizeCountTask is possible to run more times in each defined interval? if we are calling the `process()` here too.